src/api/controllers/IdentityController.js

```diff
@@ -110,6 +110,97 @@ function getHashedMessage(identity, owner, type) {
     return hashedMessage;
 }
 
+/**
+ * Looks for the user identity.
+ * @returns {Promise<Identity>} An object with the identity information.
+ *
+ * Identity can be:
+ *   - a Solana (SNS) domain (`.sol`)
+ *   - an Ethereum (ENS) domain (`.eth`)
+ *   - a Point identity (registered in Point Identity Contract)
+ *
+ * The priority to choose an identity (should multiple be available) is:
+ *   - SNS
+ *   - ENS
+ *   - Point
+ *
+ * If you are interested in one source of identity only, set a `targetOnly`:
+ *   - `targetOnly === 'solana'` -> identity is only searched for in SNS.
+ *   - `targetOnly === 'ethereum'` -> identity is only searched for in ENS and Point Identity Contract.
+ */
+async function getIdentity(
+    {solAddress, ethAddress, targetOnly, solNetwork, ethNetwork} = {
+        solAddress: '',
+        ethAddress: '',
+        targetOnly: '',
+        solNetwork: 'solana',
```

Comment:
What should we do when it comes to connecting to Solana?
 
For now, the default here is to use solana mainnet, but for that work we would need to uncomment the solana provider information in config/default.yaml. The problem is that we would be using a mainnet in our ynet, and I don't know if we want to do that.

I tried using Bonfida (for SNS stuff) with solana_devnet and solana_testnet but it doesn't work. I asked them some time ago and they said they only supported mainnet.